packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +397,19 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, if this is a chat input command, this automatically returns the command's mention instead of the ApplicationCommand object.
```

Comment:
This line is too long, eslint/prettier isn't going to like it
```suggestion
   * When concatenated with a string, if this is a chat input command,
   * this automatically returns the command's mention instead of the ApplicationCommand object.
```